projects/collection-editor-library/src/lib/components/question-option-sub-menu/question-option-sub-menu.component.ts

```diff
@@ -1,4 +1,6 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { EditorService } from '../../services/editor/editor.service';
```

Comment:
Why this service imported and not used?

Reply:
we have used in the question-option-sub-menu.html for disabling the input if the mode is review